pkgs/development/python-modules/esper/default.nix

```
@@ -0,0 +1,20 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  flit-core
+}:
+buildPythonPackage rec {
+  pname = "esper";
+  version = "3.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "benmoran56";
+    repo = "esper";
+    rev = "v${version}";
+    hash = "sha256-oH3POxrQPge9BZHG5Y/n9/sWjiCA19mqpoCZfPM6BzA=";
+  };
+
+  build-system = [flit-core];
+  doCheck = false;
```

Upstream repo contains [tests](https://github.com/benmoran56/esper/tree/v3.2/tests).

```suggestion
  doCheck = false;
  
  pythonImportsCheck = [
    "esper"
  ];  
```

---

> Upstream repo contains [tests](https://github.com/benmoran56/esper/tree/v3.2/tests).

Looks like that `pytestCheckHook` will work.

```nix
  nativeCheckInputs = [
    pytestCheckHook
  ];
```